services/judge/judge_test.go

```diff
@@ -0,0 +1,68 @@
+package judge_service
+
+import (
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/gin-gonic/gin"
+	"github.com/oj-lab/oj-lab-platform/models"
+	judge_model "github.com/oj-lab/oj-lab-platform/models/judge"
+	problem_model "github.com/oj-lab/oj-lab-platform/models/problem"
+	gorm_agent "github.com/oj-lab/oj-lab-platform/modules/agent/gorm"
+)
+
+func TestGetJudge(t *testing.T) {
+	db := gorm_agent.GetDefaultDB()
+	problem := &problem_model.Problem{
+		Slug: "test-judge-service",
+	}
+	var err error
+	err = problem_model.CreateProblem(db, *problem)
+	if err != nil {
+		t.Error(err)
```

Comment:
`t.Error` won't stop the test process, since the following everything will fail, `t.Fatal` maybe more suitable in this case.